packs/extinction-curse-bestiary/xilvirek.json

```diff
@@ -560,7 +560,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The xilvirek has two tentacles on its back that it can use to suck the life from its prey. A creature that starts its turn @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] by the xilvirek becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1}, and the xilvirek regains [[/r 20[healing]]]{20 HP}. The drained value increases by 1 on each subsequent round that the creature starts its turn grabbed, but the xilvirek doesn't regain additional HP from draining the same creature more than once per day.</p>"
+                    "value": "<p>The xilvirek has two tentacles on its back that it can use to suck the life from its prey. A creature that starts its turn @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] by the xilvirek becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1}, and the xilvirek regains @Damage[20[healing]]{20 HP}. The drained value increases by 1 on each subsequent round that the creature starts its turn grabbed, but the xilvirek doesn't regain additional HP from draining the same creature more than once per day.</p>"
```

Comment:
```suggestion
                    "value": "<p>The xilvirek has two tentacles on its back that it can use to suck the life from its prey. A creature that starts its turn @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] by the xilvirek becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1}, and the xilvirek regains @Damage[20[healing]] HP. The drained value increases by 1 on each subsequent round that the creature starts its turn grabbed, but the xilvirek doesn't regain additional HP from draining the same creature more than once per day.</p>"
```